packages/app/components/Notification/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { type FC } from 'react';
```

Comment:
put this file, and other "modal" type of component - under `packages/app/modals`

Reply:
and name it use convention: `NotificationModal` instead of just Notification (like `SomethingFeature` for feature).